js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -735,12 +735,6 @@ isBinaryAttributeHelp=Sollte für binäre LDAP-Attribute aktiviert werden.
 passwordPolicyHintsEnabled=Passwortrichtlinien-Hinweise aktiviert
 passwordPolicyHintsEnabledHelp=Gilt nur für beschreibbares MS Active Directory (MSAD). Wenn diese Option aktiviert ist, wird beim Aktualisieren des Passworts eines MSAD-Benutzers die LDAP_SERVER_POLICY_HINTS_OID-Erweiterung verwendet, was bedeutet, dass erweiterte MSAD-Passwortrichtlinien wie "Passworthistorie" oder "minimales Passwortalter" angewendet werden. Diese Erweiterung funktioniert nur für MSAD 2008 R2 oder neuer.
 managePriorities=Prioritäten verwalten
-alwaysReadEnabledValueFromLdap="Benutzer Aktiv" Status immer aus LDAP lesen
-alwaysReadEnabledValueFromLdapHelp=Wenn diese Option aktiviert ist, wird der Aktivierungs-/Deaktivierungsstatus des Benutzers immer aus MSAD gelesen, indem der entsprechende userAccountControl Wert überprüft wird.
-alwaysReadEnabledValueFromLdapLds="Benutzer Aktiv" Status immer aus LDAP lesen
-alwaysReadEnabledValueFromLdapLdsHelp=Wenn eingeschaltet, wird der Aktivierungs-/Deaktivierungsstatus des Benutzers immer aus der MSAD-LDS gelesen, indem der Wert msDS-UserAccountDisabled überprüft wird.
-ldapDerFormatted=DER Formatiert
-ldapDerFormattedHelp=Aktivieren Sie diese Option, wenn das Zertifikat in LDAP DER-formatiert ist und nicht PEM-formatiert.
```

Comment:
@thomasdarimont - I noticed that you added those properties recently, but they don't appear in the English properties file, Weblate now removed them again. If they are relevant, add them in a PR to the main repository first, and then translate them in Weblate again.

Reply:
Ah I see what I did there... good catch. While I was working on the translations I noticed that quite a lot of  labels and descriptions of Mapper/Component configuration properties are hardcoded in their respective classes, e.g.: org.keycloak.storage.ldap.mappers.msadlds.MSADLDSUserAccountControlStorageMapperFactory.

I then tried to replace the hardcoded strings with proper translation Keys and added them to the message_en.properties and message_de.properties. Later I reverted (stashed) the change but apparently I forgot to remove the translation keys.

We can remove the "unbound" keys from the PR. I'll create another issue to track how to translate the configurable settings defined via `ProviderConfigProperty`'s.

Reply:
They have already been removed :-) 

When you create a PR to make them translatable, create only the English message and the Java code changes. After the PR has been merged, translate the new keys once they appear in Weblate.